### PR TITLE
[Discussion] Arena 2024 Roadmap.

### PR DESCRIPTION
**Pull request by Syulin7**
I have updated the Arena 2024 roadmap.

### CODE REVIEW

**Comment by Syulin7** [comment]
@cheyang @AlanFokCo PTAL, please let me know your thoughts, thanks!

File: ROADMAP.md [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
 
 This document defines a high level roadmap for Arena development.
 
-### 2019
+* Objective：Simplify the user experience by deeply integrating with the Kubeflow Ecosystem.
+    * Kubeflow Integration
+        * Prepare Arena for  release v1.0.0 alongside kubeflow v1.9.0.
+        * Develop a seamless integration with the Training Operator to help simplify model training using command line.
+        * Integrate with Kubeflow Pipelines to facilitate model training from a Pipeline.
+        * Enable mode serving with KServe.
```
Comment (cheyang) [inline ROADMAP.md:12]: I think we can also consider Katib integration.

**Comment by cheyang** [comment]
Thanks very much for creating the roadmap. It's great to me. @Syulin7

File: ROADMAP.md [hunk 2]
```diff
@@ -1,10 +1,39 @@
...
+    * Integrate [Fluid project](https://github.com/fluid-cloudnative/fluid) for efficient data management.
+    * Add support for Ray Job management with [Kuberay](https://github.com/ray-project/kuberay).
+
+* Objective: Boost Performance and Stability.
+    * Regularly publish recommended practices documentation.
+    * Enhance code quality by:
+        * Reduce repetitive code.
+        * Enhance unit test.
+    * Implement automated End-to-End Test:
+        * Execute against an ACK Cluster.
```
Comment (cheyang) [inline ROADMAP.md:31]: How about also adding test in GKE?

**Comment by AlanFokCo** [comment]
> @cheyang @AlanFokCo PTAL, please let me know your thoughts, thanks!

Thanks very much for creating the roadmap.

**Comment by TrafalgarZZZ** [comment]
How about making arenasdk more user-friendly to submit deep learning training jobs? Plus, I think it would be great if arena supports one-click serving endpoint deployment with `arena serve`?

**Comment by Syulin7** [comment]
> How about making arenasdk more user-friendly to submit deep learning training jobs? Plus, I think it would be great if arena supports one-click serving endpoint deployment with `arena serve`?

@TrafalgarZZZ  Great, make sense.

File: ROADMAP.md (hunk 2 above)
Comment (Syulin7) [reply ROADMAP.md:31]: Done

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**